ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/responsiveGrid.html

```diff
@@ -26,7 +26,7 @@
          title="${panel.tooltipVisible ? '' : panel.tooltipText}"
          role="${container.roleAttribute}"
          style="${container.backgroundStyle @ context='styleString'}">
-        <div class="cmp-container__label-container">
+        <div class="cmp-container__label-container" role="heading">
```

Comment:
If you are adding the `role="heading"` attribute, you must also include the `aria-level="1"` to aria-level="6" attribute, depending on the appropriate heading level required. In this scenario, we should add `aria-level="2"`.